cvmfs/cvmfs_config

```diff
@@ -279,11 +279,13 @@ cvmfs_setup() {
     fi
   fi
 
-  # if group fuse exists, add user cvmfs
```

Comment:
In order to stay consistent with previous behavior, I'll revert the change in this block, so if a cvmfs user exists already it is still added to the fuse group. There is no error here if no user exists.